### PR TITLE
chore: clean up AES app before upgrading to arm2

### PR DESCRIPTION
**Pull request by akomisarek**
### Change description ###

Clean up old AES app to make upgrade to AzureRM2 easier. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: infrastructure/aat.tfvars [hunk 1]
```diff
@@ -1,9 +1 @@
-capacity = "2"
-idam_api_url = "https://idam-api.aat.platform.hmcts.net"
-idam_client_redirect_uri = "https://rpe-bulk-scan-processor-sandbox.service.core-compute-sandbox.internal/oauth2/callback"
 supported_jurisdictions = ["SSCS", "BULKSCAN", "DIVORCE", "PROBATE", "CMC"]
```
Comment (mdayican) [inline infrastructure/aat.tfvars:1]: why do you keep these?

File: infrastructure/main.tf [hunk 2]
```diff
@@ -42,62 +35,7 @@ locals {
...
-                                    formatlist("IDAM_USERS_%s_USERNAME", keys(local.supported_users)),
-                                    data.azurerm_key_vault_secret.idam_users_usernames.*.value
-                                )}"
-
-  users_passwords_settings = "${zipmap(
-                                    formatlist("IDAM_USERS_%s_PASSWORD", keys(local.supported_users)),
-                                    data.azurerm_key_vault_secret.idam_users_passwords.*.value
-                                )}"
-
   s2s_rg  = "rpe-service-auth-provider-${local.local_env}"
```
Comment (mdayican) [inline infrastructure/main.tf:38]: why are we keeping this?

**Review by aliveni — APPROVED** [review]

File: infrastructure/aat.tfvars (hunk 1 above)
Comment (akomisarek) [reply infrastructure/aat.tfvars:1]: It's used to build following vault keys:idam_users_usernames / idam_users_passwords, ie idam-users-cmc, etc

If that's not used later I am happy to remove but probably it's going to delete those keys from vault!

File: infrastructure/main.tf (hunk 2 above)
Comment (akomisarek) [reply infrastructure/main.tf:38]: It's used to build following vault keys:idam_users_usernames / idam_users_passwords, ie idam-users-cmc, etc

**Pull request merged**